test/Microsoft.AspNet.Mvc.ViewFeatures.Test/ViewFeatures/DefaultDisplayTemplatesTest.cs

```diff
@@ -122,12 +122,10 @@ public void ObjectTemplateDisplaysNullDisplayTextWhenObjectIsNull()
         public void ObjectTemplate_IgnoresPropertiesWith_ScaffoldColumnFalse()
         {
             // Arrange
-            var expected =
-@"<div class=""HtmlEncode[[display-label]]"">HtmlEncode[[Property1]]</div>
-<div class=""HtmlEncode[[display-field]]""></div>
-<div class=""HtmlEncode[[display-label]]"">HtmlEncode[[Property3]]</div>
-<div class=""HtmlEncode[[display-field]]""></div>
-";
+            var expected = "<div class=\"HtmlEncode[[display-label]]\">HtmlEncode[[Property1]]</div>" + Environment.NewLine +
+"<div class=\"HtmlEncode[[display-field]]\"></div>"+Environment.NewLine +
+"<div class=\"HtmlEncode[[display-label]]\">HtmlEncode[[Property3]]</div>"+Environment.NewLine+
+"<div class=\"HtmlEncode[[display-field]]\"></div>"+Environment.NewLine;
             var model = new DefaultTemplatesUtilities.ObjectWithScaffoldColumn();
```

Comment:
Strange indentation now that you're not using a here string. Also try to get into the habit of putting spaces around binary operators even in test code. That way you won't mess up the product code.